src/Raven.Server/Documents/Indexes/MapReduce/Auto/ReduceMapResultsOfAutoIndex.cs

```diff
@@ -52,6 +52,9 @@ protected override AggregationResult AggregateOn(List<BlittableJsonReaderObject>
                                     if (obj.TryGetMember(propertyName, out value) == false)
                                         throw new InvalidOperationException($"Could not read numeric value of '{propertyName}' property");
 
+                                    if (value == null)
```

Comment:
It is fine to aggregate on null, but it isn't fine to aggregate on non existing field?
Not sure how we are suppose to behave here, ideas?

Reply:
What do you mean by "It is fine to aggregate on null"? It isn't, that why we throw this exception.

It doesn't matter if we a field is null or it doesn't exist - we cannot process `sum()` on such field so we'll throw.

Reply:
I'm trying to think as the user. If the value is null or missing, can we just evaluate it as zeo?

Reply:
Since `new List<int?> { null, 1 }.Sum()` evaluates to `1` then we could behave the same and treat `null` or missing field as a `0`. That would be consistent with the behavior of static map-reduce indexes.

Reply:
Yes, that is what I would expect.